portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Projects</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+      <div class = "content">
+          <h1> Past Projects </h1>
+          <nav>
+          <ul id = "horizontal-list">
+            <li><a href= "index.html">Home </a></li>
+            <li><a href="https://www.linkedin.com/in/ali-kew">LinkedIn</a></li>
+            <li><a href="https://github.com/alikew24"> Github </a></li>
+            <li><a href="projects.hmtl"> Projects </a></li>
+            <li><a href= "hobbies.html">Hobbies </a></li>
+          </ul>
+          </nav>
+          <h3> Connect Four </h3>
```

Comment:
Headers should usually follow a hierarchy, i.e. sub-sections of `<h1>` are denoted by `<h2>` and so on.

If you're using `<h3>` here because you want a smaller font, that can be done by styling `h2` with CSS, e.g.

```css
h2 {
  font-size: 0.9em; /* play around with the actual value until it looks good to you */
}
```